decidim-core/app/views/layouts/decidim/mailer.html.erb

```diff
@@ -86,7 +86,8 @@
                       <th class="expander"></th>
                       <th class="small-12 first columns cityhall-bar">
                         <div class="decidim-logo" style="float: right; text-align: right; padding-right: 16px">
-                          <%= link_to @organization.name, decidim.root_url(host: @organization.host) %>
+                          <% url =  decidim.root_url(host: @organization.host) + "?utm_source=" + @organization.host + "&utm_campaign=" + @newsletter.id.to_s %>
```

Comment:
Move this to a helper, maybe?

Reply:
Again, we shouldn't couple `decidim` to Google Analytics.